src/scss/foundations/colors.scss

```diff
@@ -0,0 +1,132 @@
+@use "../lib";
+
+$_color-action-base: #067f96;
```

Comment:
Userei `camelCase` come notation

Reply:
Cambiati, poi ci ri-ragioniamo quando (e se) utilizzeremo i tokens, che in genere sono sempre in kebab-case